src/sentry/profiles/task.py

```diff
@@ -139,6 +139,24 @@ def _should_deobfuscate(profile: Profile) -> bool:
     return platform in SHOULD_DEOBFUSCATE and not profile.get("deobfuscated", False)
 
 
+def get_profile_platforms(profile: Profile) -> List[str]:
+    platforms = [profile["platform"]]
+
+    if "version" in profile and profile["platform"] in SHOULD_SYMBOLICATE_JS:
+        for frame in profile["profile"]["frames"]:
+            if frame.get("platform", "") == "cocoa":
+                platforms.append(frame["platform"])
```

Comment:
I wonder if this might be easier to read with an `if any(…): platforms.append("cocoa")`?

Reply:
Are you suggesting something like this?

```
if any(map(lambda f: f.get("platform","")=="cocoa", frames)):
    platforms.append(frame["cocoa"])
```

Tbh, I think it depends on how familiar with the functional style are the people reading the code. The iterative ones doesn't seem too complex neither.